schema/fmi3ModelDescription.xsd

```diff
@@ -141,45 +146,13 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 						<xs:group ref="fmi3Variable" maxOccurs="unbounded"/>
 					</xs:complexType>
 				</xs:element>
-				<xs:element name="ModelStructure">
+				<xs:element name="ModelStructure" minOccurs="0">
 					<xs:complexType>
 						<xs:sequence>
-							<xs:element name="Outputs" type="fmi3VariableDependency" minOccurs="0">
```

Comment:
Please make such cleanups in a consistent way or skip such cosmetic changes!
Before this change everything was consistent and by this cosmetic change the document is now inconsistent.
The removed container elements Output**s**, Derivative**s** and InitialUnknown**s** are still referenced at many many places in the document.